src/Cell.cpp

```diff
@@ -1041,10 +1036,47 @@ void Cell::ringify(std::vector<Cell*>* subcells) {
                "the 2 halfspaces for each Surface.", _id, halfspace1,
                zcylinder1->getId(), halfspace2, zcylinder2->getId());
 
+  /* Loop over ZCylinders and create a new Cell clone for each ring */
+  std::vector<ZCylinder*>::iterator iter2;
+  std::vector<Cell*>::iterator iter3;
+
+  /* If this is an unbounded Cell - i.e., moderator in a fuel pin cell - then
+   * make the first ring span the volume outside the outermost cylinder */
+  if (radius1 > radius2) {
+    ZCylinder* outer = new ZCylinder(x1, y1, radius1);
+
+    /* Create outermost ZCylinder for each of the sectorized Cells */
+    if (subcells->size() != 0) {
```

Comment:
Because it is passed in as an argument to this method. If it weren't a pointer, we'd need copy constructors in all of our methods (which we once had) *and* it will slow this method down a lot.